src/store/store.js

```diff
@@ -0,0 +1,8 @@
+import { configureStore } from '@reduxjs/toolkit';
+import filtersReducer from '../slices/filtersSlice';
+
+export const store = configureStore({
+  reducer: {
```

Comment:
You can also make use of this key: `devTools: process.env.NODE_ENV !== 'production'` which exposes what is going on with redux on the browser dev tools